src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -539,11 +373,11 @@ void Profile::ResetIconPath()
 // - this profile's icon path, if one is set. Otherwise returns the empty string.
 winrt::hstring Profile::GetExpandedIconPath() const
 {
-    if (!HasIcon())
+    if (_IconPath.empty())
     {
         return { L"" };

```

Comment:
Why do we do this? Can't we just return the default constructed hstring instead of passing in 0-length wchar_t* first?